lstchain/scripts/lstchain_data_muon_analysis.py

```diff
@@ -62,6 +55,11 @@
                                          "Default = 442",
                     type = int, default = 442)
 
+parser.add_argument("--tel_id", help = "telescope id"
+                                         "Default = 1",
+                    type = int, default = 1)
```

Comment:
`tel_id` should be 0 for run number < xxx, right?
We should either specify the run number and select the tel_id with it or specify the tel_id without mentioning the run number

Reply:
Yes, indeed, what do you prefer?